ddl/rollingback.go

```diff
@@ -230,12 +246,16 @@ func convertJob2RollbackJob(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job)
 		ver, err = rollingbackAddColumn(t, job)
 	case model.ActionAddIndex:
 		ver, err = rollingbackAddindex(w, d, t, job)
+	case model.ActionAddTablePartition:
```

Comment:
add model.ActionAddTablePartition and model.ActionDropTablePartition type in the admin.isJobRollbackable  function?

Reply:
Done.